scripts/run-jmh.sh

```diff
@@ -1,6 +1,6 @@
-#! /bin/bash
+#!/usr/bin/env bash
```

Comment:
It might be smoother to move some of these script improvements to their own PR. I don't normally mind drive by changes but i suspect there will be some discussion around the main purpose of the PR and don't want these improvements to sit unused